### PR TITLE
Some minor fixes to get the deployed dev working correctly

### PR DESCRIPTION
**Pull request by andrewpatto**
The esbuild process breaks DI so I am pivoting to just running the server via ts-node (same as actual local dev)

### CODE REVIEW

File: application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx [hunk 1]
```diff
@@ -46,6 +46,10 @@ export const LogsBox: React.FC<Props> = ({ releaseId, pageSize }) => {
   const baseColumnClasses = "py-2 font-small text-gray-400 whitespace-nowrap";
 
   const createRows = (data: AuditEntryType[]) => {
+    // whilst it shouldn't happen in real life - in dev we can occasionally have
+    // releases with no audit entries - so we need to handle that safely
+    if (!data) return [];
+
```
Comment (andrewpatto) [inline application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx:52]: This was causing an issue in some of my dev scenarios

**Comment by andrewpatto** [comment]
This code has actually been deployed to the current dev deployed instance

File: application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx (hunk 1 above)
Comment (williamputraintan) [reply application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx:52]: Yes, I suppose this is the property map does not exist (or something ... )?

**Review by williamputraintan — APPROVED** [review]

File: application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx (hunk 1 above)
Comment (mmalenic) [reply application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx:52]: Wouldn't the map operation below handle an empty array as well? Or does this check if the array is undefined?

Comment (williamputraintan) [reply application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx:52]: I think this checks for `undefined`, `.map` should handle OK for empty array.

Comment (mmalenic) [reply application/frontend/src/pages/releases/detail/logs-box/logs-box.tsx:52]: Ahh yeah, makes sense.

**Review by mmalenic — APPROVED** [review]

**Pull request merged**